solidity/contracts/BancorNetwork.sol

```diff
@@ -207,48 +194,34 @@ contract BancorNetwork is TokenHolder, ContractRegistryClient, ReentrancyGuard {
     /**
      * @dev converts the token to any other token in the bancor network by following
      * a predefined conversion path and transfers the result tokens to a target account
-     * affiliate account/fee can also be passed in to receive a conversion fee (on top of the liquidity provider fees)
      * note that the network should already have been given allowance of the source token (if not ETH)
      *
      * @param _path                conversion path, see conversion path format above
      * @param _amount              amount to convert from, in the source token
      * @param _minReturn           if the conversion results in an amount smaller than the minimum return - it is cancelled, must be greater than zero
      * @param _beneficiary         account that will receive the conversion result or 0x0 to send the result to the sender account
-     * @param _affiliateAccount    wallet address to receive the affiliate fee or 0x0 to disable affiliate fee
-     * @param _affiliateFee        affiliate fee in PPM or 0 to disable affiliate fee
      *
      * @return amount of tokens received from the conversion
      */
-    function convertByPath(
+    function convertByPath2(
```

Comment:
Why do we need to rename this method?

Reply:
Because it is part of the API, and we don't want to break any application which relies on this.